nextcord/shard.py

```diff
@@ -357,6 +357,7 @@ def __init__(
         rollout_register_new: bool = True,
         rollout_update_known: bool = True,
         rollout_all_guilds: bool = False,
+        default_guild_ids: Optional[List[int]] = None,
```

Comment:
Does this also need to be added to ext.commands.Bot? (I haven't tested)

Reply:
Yup

Reply:
030af98